plugins/CoreAdminHome/Commands/PurgeOldArchiveData.php

```diff
@@ -58,6 +58,8 @@ protected function configure()
         $this->addOption('exclude-invalidated', null, InputOption::VALUE_NONE, "Do not purge invalidated archive data.");
         $this->addOption('exclude-ranges', null, InputOption::VALUE_NONE, "Do not purge custom ranges.");
         $this->addOption('skip-optimize-tables', null, InputOption::VALUE_NONE, "Do not run OPTIMIZE TABLES query on affected archive tables.");
+        $this->addOption('include-year-archives', null, InputOption::VALUE_NONE, "If supplied, the command will purge archive tables that contain year archives for every supplied date.");
```

Comment:
Ok I'm not sure i'm getting it. I thought that if users have to manually purge year archives, it means the archive purger is not doing the job of purging year archives "automatically". so if we need this parameter in the console, I thought that the Scheduled Task itself which does not have this option, wouldn't properly invalidate yearly archives. But maybe i'm missing something so i may be wrong!